tests/integration_tests/with_db/test_audit_table.py

```diff
@@ -19,3 +67,56 @@ def test_create_audit_table(pyexasol_connection, db_schema, exa_all_columns):
         for c in (BaseAuditColumns.all + additional_columns)
     }
     assert actual == expected
+
+
+def test_audit_query(pyexasol_connection, audit_table, subquery_table):
```

Comment:
You need to tests for audit query, one with select and one without

Reply:
Yes. I discovered that gap as well. 
Tests will be enhanced in next push.